lib/europeana/api/record/hierarchy.rb

```diff
@@ -0,0 +1,83 @@
+require 'rubytree'
+require 'active_support/core_ext/object/blank'
+
+module Europeana
+  module API
+    class Record
+      ##
+      # Retrieve record hierarchies over the Europeana API
+      class Hierarchy < Tree::TreeNode
+        include Requestable
+
+        attr_reader :record
+
+        ##
+        # @param record [Europeana::API::Record]
+        def initialize(record)
+          @record = record
+          super(record.id, execute_request[:self])
+          fetch_family
+        end
+
+        ##
+        # Retrieves parent, children and siblings of this node
+        def fetch_family
+          unless content[:parent].blank?
+            parent_response = execute_request(rel: :parent)[:parent]
+            parent_node = Tree::TreeNode.new(parent_response[:id], parent_response)
+
+            execute_request(rel: "preceeding-siblings")["preceeding-siblings"].each do |sibling|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.